spyql/processor.py

```diff
@@ -109,7 +110,10 @@ def prepare_expression(self, expr):
 
         return [self.strings.put_strings_back(expr)]
 
-    def compile_clause(self, clause, clause_modifier = None, single = True, mode = 'eval'):
+    def is_clause_single(self, clause):
+        return clause not in ['select']
```

Comment:
:+1: Good job avoiding the boolean parameter in `compile_clause()` and `eval_clause()`